tests/hikari/impl/test_special_endpoints.py

```diff
@@ -711,6 +724,8 @@ async def test_create(self):
             guild=undefined.UNDEFINED,
             default_permission=builder.default_permission,
             options=builder.options,
+            name_localizations=undefined.UNDEFINED,
+            description_localizations=undefined.UNDEFINED,
```

Comment:
Here you probably want to set them to something and make sure they get passed correctly